src/Native/Unix/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ if (NOT WIN32)
         # Ensure that objcopy is present
         if(DEFINED ENV{CROSSCOMPILE})
             if(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l OR CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL i686)
-                find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
+                find_program(OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
```

Comment:
I don't see the existing cross compilation support setting TOOLCHAIN_PREFIX:
https://github.com/dotnet/corefx/blob/68571734f84db4cebce5356bb3138569252caf19/cross/armel/toolchain.cmake#L8
Is this going to break those?